index.js

```diff
@@ -115,12 +117,26 @@ AnnotationPoller.prototype._applyReplacements = function (obj) {
       if (row.image) {
         if (row.image.url) row.image.url = _this._escape(row.image.url)
       }
+
+      // flatten the row object into an ordered
+      // set of elements. In our template we output
+      // the elements in the order that the keys appear.
+      Object.keys(row).forEach(function (key) {
```

Comment:
Given the complexity of the transform, I'm inclined to do it outside the template. I'd like the implementers to continue using the existing format:

```js
{
  rows: [
   {
      image: {},
      link: {}
   }
  ]
}
```

which then gets flattened into an array, and iterated over ... allowing the order of the keys in the object to have an effect on the order that elements are displayed in the addon.